### PR TITLE
Feat/next image

### PR DESCRIPTION
**Pull request by prodvdc**

### CODE REVIEW

File: components/Icons/CoinIcon/index.js [hunk 1]
```diff
@@ -10,26 +11,21 @@ const useStyles = makeStyles(() => ({
     display: 'flex'
   },
   img: props => ({
-    height: props.height,
+    height: props.size,
```
Comment (saito-sv) [inline components/Icons/CoinIcon/index.js:14]: `size` usually represents  2 values: `width, height` but it seems that you are using it for presenting `height` only. I believe the use of `size` here is miss-leading, unless am missing some internal functionality.

Comment (prodvdc) [reply components/Icons/CoinIcon/index.js:14]: image
The "size" is used as width and height props values for Image component.
@saito-sv

**Review by saito-sv — CHANGES_REQUESTED** [review]
Am ok with the changes in the UI but I cant accept the changes in our package-lock file, only core devs are allowed to upgrade our packages. If you ever feel the need to a package to be updated please report to @Jonasslv  or myself @saito-sv

**Comment by saito-sv** [comment]
@prodvdjin  Also, please make sure you based off your working branch from development and when creating a PR, make sure your commit history is clean by always rebasing to latest.

**Comment by ghost** [comment]
Closed because you submitted #260

**Pull request closed without merging**